nautobot_golden_config/tests/test_views.py

```diff
@@ -0,0 +1,47 @@
+"""Unit tests for nautobot_golden_config views."""
+
+from unittest import skip
```

Comment:
Are you using this? Believe this is the pylint error you're seeing.

Reply:
@scetron see this comment?

Reply:
I did now 😅